### PR TITLE
Ng Update

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: packages/schematics/migrations/update-6-0-0/update-6-0-0.ts [hunk 1]
```diff
@@ -289,14 +290,22 @@ function deleteUnneededFiles(host: Tree) {
...
-function patchLibIndexFiles(host: Tree) {
+function patchLibIndexFiles(host: Tree, context: SchematicContext) {
   const angularJson = readJsonInTree(host, 'angular.json');
 
   Object.values(angularJson.projects).forEach((p: any) => {
     if (p.projectType === 'library') {
       fs.renameSync(p.sourceRoot, `${p.root}/lib`);
       fs.mkdirSync(p.sourceRoot);
       fs.renameSync(`${p.root}/lib`, `${p.sourceRoot}/lib`);
+      const npmScope = readJsonInTree(host, 'nx.json').npmScope;
```
Comment (vsavkin) [inline packages/schematics/migrations/update-6-0-0/update-6-0-0.ts]: Use getNpmScope

File: packages/schematics/migrations/update-6-0-0/update-6-0-0.ts [hunk 2]
```diff
@@ -289,14 +290,22 @@ function deleteUnneededFiles(host: Tree) {
...
+      host = updateJsonInTree('tsconfig.json', json => {
```
Comment (vsavkin) [inline packages/schematics/migrations/update-6-0-0/update-6-0-0.ts]: either rewrite it with read and override, or rewrite the whole patchLibIndexFiles as a chain of rules.  

This looks a bit convoluted for updating a single field.

**Comment by vsavkin** [comment]
merged here https://github.com/nrwl/nx/commit/8de81d6c16935393239b2d7093bc8e72ab251311

**Comment by juristr** [comment]
Awesome, are you planning to create a new RC out of it? I'm currently in the middle of upgrading (partially manually) my NX workspace since today morning. could totally test this out whether it works in my monorepo (with 3 apps and a couple of libs)

**Comment by juristr** [comment]
nevermind, found it 😅

**Pull request closed without merging**